src/commands/lambda-link-to-application.ts

```diff
@@ -0,0 +1,48 @@
+import {Command} from '@commander-js/extra-typings';
+import {FusionAuthClient, ApplicationRequest} from '@fusionauth/typescript-client';
+import chalk from 'chalk';
+import {errorAndExit, getApplicationName} from '../utils.js';
+import {apiKeyOption, hostOption} from "../options.js";
+
+const action = async function ( applicationId: string,
+                                lambdaId: string,
+                                {key: apiKey, host}:
+                                    {
+                                        key: string;
+                                        host: string
+                                    }
+                                ): Promise<void>
+{
+    console.log(`Linking lambda ${lambdaId} to application ${applicationId} on ${host}`);
+    try {
+        const applicationName = await getApplicationName(applicationId, {key: apiKey, host});
+        const request: ApplicationRequest = {
+            "application": {
+                "name": applicationName,
+                "lambdaConfiguration": {
+                    "accessTokenPopulateId": lambdaId,
+                    "idTokenPopulateId": lambdaId
+                }
+            }
+        };
+        const fusionAuthClient = new FusionAuthClient(apiKey, host);
+        const clientResponse = await fusionAuthClient.updateApplication(applicationId, request)
```

Comment:
This is dangerous. As the [API documentation](https://fusionauth.io/docs/v1/tech/apis/applications#update-an-application) states, with PUT / `updateApplication` you must specify all of the properties of the Application. This will cause data loss in the application configuration.

Use `patchApplication` instead, then you can also drop the `name` property in the request.